src/modules/QuickLinks/QuickLinks.module.css

```diff
@@ -0,0 +1,63 @@
+@import '_colors.css';
+@import '_fonts.css';
+
+@define-mixin list-content {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: $color-text-secondary;
+  font-weight: 500;
+  font-size: $medium-font-size;
+  line-height: 1.75rem;
+}
+
+.quick-links {
+  margin-top: 1rem;
+  padding: 0.3rem;
+
+  &__heading {
+    padding: 0.6rem;
+    color: #323639;
```

Comment:
lest use constant for colors

Reply:
sure, didn't know what to name :) can you suggest please